cheatsheets/Kubernetes_Security_Cheat_Sheet.md

```diff
@@ -380,16 +380,16 @@ When you are configuring the security context for your pods, only grant the priv
 
 Security Context Settings:
 
-1. SecurityContext->runAsNonRoot
+1. Security Context Setting: SecurityContext->**runAsNonRoot**  

```

Comment:
In all of these cases, I don't see why we need the "Security Context Setting: " prefix. I figure you added this one for consistency, but I think we're better off removing it from the other ones instead.

Reply:
I totally agree. If it's okay with everyone, I would like to remove the "Security Context Setting:" prefix.